packages/core/src/App/Containers/RealAccountSignup/account-wizard.jsx

```diff
@@ -280,7 +280,7 @@ class AccountWizard extends React.Component {
                             </div>
                         )}
                     </DesktopWrapper>
-                    <Div100vhContainer className='account-wizard__body' is_disabled={isDesktop()} height_offset='40px'>
+                    <Div100vhContainer className='account-wizard__body' is_disabled={isDesktop()} height_offset='110px'>
```

Comment:
Any reason why the `offset` was increased? To my knowledge for the `AccountWizard` containers, we only offset the fullscreen-dialog's mobile header height which is `40px`. Would this have any affect on normal `RealAccountSignup` screens?

Reply:
Yeah, so, `header-steps` is no longer sticking to the top via position-absolute. so we don't have to do `margin-top: 7rem` and stuff like that. I've just increased the offset here.